openrtb_ext/imp_openweb.go

```diff
@@ -2,8 +2,7 @@ package openrtb_ext
 
 // ExtImpOpenWeb defines the contract for bidrequest.imp[i].ext.prebid.bidder.openweb
 type ExtImpOpenWeb struct {
-	SourceID    int     `json:"aid"`
-	PlacementID int     `json:"placementId,omitempty"`
-	SiteID      int     `json:"siteId,omitempty"`
-	BidFloor    float64 `json:"bidFloor,omitempty"`
+	Aid         int    `json:"aid"`
+	Org         string `json:"org"`
+	PlacementID string `json:"placementId,omitempty"`
```

Comment:
`PlacementID` is marked as required in static/bidder-params/openweb.json

Therefore this field cannot be omitempty